src/Marketplace/ProduceCard.js

```diff
@@ -57,16 +60,16 @@ const styles = StyleSheet.create({
 });
 
 function ProduceCard({
-  navigation, favorited, image, name, price, unit,
+  navigation, produceId, favorited, image, name, price, unit, seller, maxQuantity,
 }) {
+  const [favorite, setFavorite] = useState(favorited);
+
   const onPressCard = () => {
     navigation.navigate('ProduceDetails', {
-      image, name, price, unit,
+      produceId, favorite, setFavorite, image, name, price, unit, seller, maxQuantity,
```

Comment:
There may be a way to avoid the warning by using navigation params and passing data back to the previous screen instead of passing setFavorite. Take at look at the documentation here to see if it helps: https://reactnavigation.org/docs/params/#passing-params-to-a-previous-screen

Reply:
reason this was not done properly:
- each heart button needs to make an individual call to airtable to update the "favorites" array on "Users UPDATED" table
- marketplace needs to call "Produce" table and then "Users UPDATED" table, followed by checking the user's favorites produce and updating "Favorited field" which was added in the `produceList` variable